scripts/analysis_2/0_download_data.R

```diff
@@ -0,0 +1,87 @@
+## Setup environment for recount
+## Only run once
+install.packages("BiocManager")
+install.packages("RMySQL")
+BiocManager::install('GenomicFeatures')
+BiocManager::install('recount')
+
+## About
+# recount2 contains over 70,000 uniformly processed human RNA-seq samples spanning
+# TCGA, SRA and GTEx
+library('recount')
+source("~/Documents/Repos/Batch_effects_simulation/scripts/functions/calculations.R")
+
+## Metadata
+
+# Download the metadata for all samples in recount, and write it to a file
+metadata <- all_metadata()
+write.table(metadata, '~/Documents/Repos/Batch_effects_simulation/data/metadata/recount2_metadata.tsv', sep='\t', row.names=FALSE)
+
+# Format of the metadata
+# Based on definitions from NCBI and blog post: 
+# https://www.ncbi.nlm.nih.gov/books/NBK56913/#search.what_do_the_different_sra_accessi
+# https://www.ccdatalab.org/blog/2019/3/29/gene-expression-repos-explained
+# Project: A sequencing study (i.e. GTEx).    
+# Sample: Physical biospecimen on which sequencing was performed, biological source material (i.e. HeLa cell line).  
+#         A project contains many samples
+# Experiment: Unique sequencing library for a specific sample.  A sample can have multiple experiments, most have 1 experiment.
+# Run: Sequencing run.  An experiment can contain many runs (i.e. technical replicates)
+# In this case, we want to group runs into projects (for "experiment-level" simulation)
+project_ids <- unique(metadata$project)
+
+# Entire recount2 is 8TB
+# We will only select the top 10 studies instead
+selected_project_ids <- sample(project_ids, 50)
```

Comment:
> We will only select the top 10 studies instead

^Your comment says select top 10 but code reads sample fifty project ids. Am I reading this correctly?

Reply:
This is a typo, I started with just 10 when i was first writing these scripts to test.  I will change this